osgi.enroute.web.simple.provider/src/osgi/enroute/web/server/provider/RedirectServlet.java

```diff
@@ -0,0 +1,74 @@
+package osgi.enroute.web.server.provider;
```

Comment:
The implementation was extremely opinionated regarding redirects. I think that by pulling out redirects and putting into a ConditionalServlet like this, it provides a more elegant and configurable solution.

In my case, I do not want *any* redirects, so by configuring to the empty string, it is an easy and elegant solution for me.